src/Compilers/Core/Portable/Compilation/Expression.cs

```diff
@@ -333,17 +333,24 @@ public void Accept(OperationVisitor visitor)
 
     internal sealed class Binary : IBinaryOperatorExpression
     {
-        public Binary(BinaryOperationKind binaryOperationKind, IOperation left, IOperation right, ITypeSymbol resultType, SyntaxNode syntax)
+        public Binary(
+            BinaryOperationKind binaryOperationKind,
+            IOperation left, IOperation right, 
+            ITypeSymbol resultType, 
+            SyntaxNode syntax, bool isLifted)
```

Comment:
i preferred this approach because it made using named-arguments later much easier...  But if you prefer it next to the kind, i can oblige :)